app/code/Magento/UrlRewriteGraphQl/Model/Resolver/UrlRewrite.php

```diff
@@ -63,8 +73,19 @@ public function resolve(
         $result = function () {
             return null;
         };
+        
         if (isset($args['url'])) {
-            $urlRewrite = $this->findCanonicalUrl($args['url']);
+            $url = $args['url'];
```

Comment:
Please add GraphQL API functional test to cover the fix.

Reply:
I will add it, thanks for feedback